index.js

```diff
@@ -113,13 +124,7 @@ module.exports = {
         return pkg.version
       },
 
-      //temporary!
-      _flumeUse                :
-        function (name, flumeview) {
-          ssb.use(name, flumeview)
-          return ssb[name]
-        },
-
```

Comment:
Removing `_flumeUse` is a big deal, right? A lot of plugins use it, so this is a breaking change? Do tests fail? And I never really understood why was it marked "temporary" and why does it start with an underscore.

Reply:
Yeah this should go back in -- this PR is super messy because it was meant to open the discussion with a hacky prototype that I wrote. I think that *really*, this PR should be:

- A new method added to `index.js` that creates a stream of sequence numbers
- A few tests added to ensure that the sequence stream is working as intended

Currently this PR has a bunch of experimental code running client-side flumeviews and stuff, but I don't think we actually want that in ssb-db.